network.sh

```diff
@@ -148,6 +148,11 @@ elif [[ "$IP_STACK" = "v4v6" || "$IP_STACK" = "v6v4" ]]; then
   export SERVICE_SUBNET_V4=${SERVICE_SUBNET_V4:-"172.30.0.0/16"}
   export SERVICE_SUBNET_V6=${SERVICE_SUBNET_V6:-"fd02::/112"}
   export NETWORK_TYPE=${NETWORK_TYPE:-"OVNKubernetes"}
+  # Although v6-primary clusters have access to both v4 and v6, during provisioning
+  # the nodes only get v6 addresses so we need to mirror images or they can't pull.
+  if [[ "$IP_STACK" = "v6v4" ]]; then
+    export MIRROR_IMAGES=true
```

Comment:
Can we use `${MIRROR_IMAGES:-true}` here? like L142? So that we only set `true` if the user did not already set `MIRROR_IMAGES:-false` in config.